plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/BannedElementTest.java

```diff
@@ -29,6 +32,38 @@ public void testBannedElementConstructor_nullElementName() {
   public void testBannedElementConstructor_nullLocation() {
     new BannedElement("test", "org.eclipse.core.resources.problemmarker", 1, null, 0);
   }
+  
+  @Test
+  public void testEquals() {
+    BannedElement element1 = new BannedElement("message");
+    BannedElement element2 = new BannedElement("message");
+    assertTrue(element1.equals(element2) && element2.equals(element1));
```

Comment:
Better to break this into two asserts so it's obvious which one failed if either did.